core/swift51Action/Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN curl -sL \
 FROM swift:5.1.2
 
 # select the builder to use
-ARG GO_PROXY_BUILD_FROM=source
+ARG GO_PROXY_BUILD_FROM=release
```

Comment:
I approved this although for "latest" nightly builds we might want to keep the source mode.

Reply:
I had my Apache release hat on and was figuring we should build releases from releases.
But I can see the nightly from nightly point too.

Reply:
I suppose that we can get both with docker args on the command line. So it’s a matter of picking the default here and release is fine I suppose.